config/samples/osp-director_v1beta1_openstackclient.yaml

```diff
@@ -6,5 +6,23 @@ metadata:
 spec:
   imageURL: quay.io/openstack-k8s-operators/tripleo-deploy
   deploymentSSHSecret: osp-controlplane-ssh-keys
+  ospNetwork:
+    name: osp
+    bridgeName: br-osp
```

Comment:
One thing I've been wondering as I've worked on the SRIOV integration is what happens when this common `Network` type (whether here in `OpenStackClient` or in `OpenStackVmSet`) conflicts based on `name`, `bridgeName` and `desiredState`.  What I mean is: what if some used the same `name` and `bridgeName` across two definitions (and the same `nodeSelector`, I suppose), but they had differing `desiredState`?  If this is a problem, should we maybe add webhooks to validate?

Reply:
yes that as a valid point and we should add it. we could do it in a follow up. this PR is intended to provide 1:1 transition from what we have atm.

Reply:
I agree.  Not needed in this PR.  I will add a task for it to the project.